packages/e2e-test-utils/src/customFonts.js

```diff
@@ -35,20 +77,19 @@ export const addCustomFont = async (fontUrl) => {
 
   await page.keyboard.type(fontUrl);
   await expect(page).toClick('button', { text: 'Add Font' });
-  // TODO(#10878): Target font by font name.
-  await page.waitForSelector('[aria-label="Remove font"]');
+  await page.waitForTimeout(300);
```

Comment:
What is the timeout for?

It's preferable to wait for a specific event or selector instead of magic timeout numbers (e.g. network response, element appearing, element changing state, etc.)

Reply:
Agreed --- just thinking about this.

Maybe we pass in the font name as well and use --- I don't really want to add another param but it would ensure the font has been added.


```javascript
await page.waitForSelector(`[aria-label="Delete ${fontName}"]`);
```

Reply:
Removed --- seems to be working.